apollo-portal/src/main/resources/static/config/diff.html

```diff
@@ -0,0 +1,153 @@
+    <link rel="stylesheet" type="text/css" href="../styles/common-style.css">
+    <title>比较配置</title>
+    <style>
+        .comment-toggle {
+            margin-left: 8px !important;
+        }
+        .diff-content {
+            margin-top: 12px;
+        }
+    </style>
+</head>
+
+<body>
+
+<apollonav></apollonav>
+
+<div class="container-fluid apollo-container" ng-controller="DiffItemController">
+    <section class="panel col-md-offset-1 col-md-10">
+        <header class="panel-heading">
+            <div class="row">
+                <div class="col-md-7">
+                    <h4 class="modal-title">比较配置
+                        <small ng-show="syncItemStep == 1">(第一步:选择比较信息)</small>
+                        <small ng-show="syncItemStep == 2">(第二步:查看差异配置)</small>
+                    </h4>
+                </div>
+                <div class="col-md-5 text-right">
+                    <button type="button" class="btn btn-primary" ng-show="syncItemStep > 1 && syncItemStep < 3"
+                            ng-click="syncItemNextStep(-1)">上一步
+                    </button>
+                    <button type="button" class="btn btn-primary" ng-show="syncItemStep < 2"
+                            ng-click="diff()">下一步
+                    </button>
+                    <button type="button" class="btn btn-info" data-dismiss="modal"
+                            ng-click="backToAppHomePage()">返回到项目首页
+                    </button>
+                </div>
+            </div>
+        </header>
+        <div class="panel-body">
+            <div class="row" ng-show="syncItemStep == 1">
+                <div class="alert-info alert no-radius">
+                    <strong>Tips:</strong>
+                    <ul>
+                        <li>通过比较配置功能，可以查看多个环境、集群间的配置差异</li>
+                    </ul>
+                </div>
+                <div class="form-horizontal">
+                    <div class="form-group">
+                        <label class="col-sm-2 control-label">要比较的集群</label>
+                        <div class="col-sm-6">
+                            <apolloclusterselector apollo-app-id="pageContext.appId" apollo-default-all-checked="false"
+                                                   apollo-select="collectSelectedClusters"
+                                                   apollo-default-checked-env="pageContext.env"
+                                                   apollo-default-checked-cluster="pageContext.clusterName"></apolloclusterselector>
+                        </div>
+                    </div>
+                </div>
+                <hr>
+            </div>
+
+            <!--step 2-->
+            <div class="row" ng-show="syncItemStep == 2">
+                <div class="row" style="margin-top: 10px;">
+                    <div class="form-horizontal">
+                        <div class="col-sm-12">
+                            <label class="col-sm-2 control-label">是否比较注释</label>
```

Comment:
This style looks a little strange, as the screen shot below

![image](https://user-images.githubusercontent.com/837658/53696238-12edb080-3e00-11e9-88cd-ad278af17e75.png)

With the following change, it may look a little better..

```html
                        <div class="col-sm-12">
                            <label class="control-label">
                                <input type="checkbox" class="comment-toggle" ng-checked="showCommentDiff"
                                       ng-click="showCommentDiff=!showCommentDiff">
                                是否比较注释
                            </label>
                        </div>
```
![image](https://user-images.githubusercontent.com/837658/53696228-f6ea0f00-3dff-11e9-91e9-2b8a38a1dda0.png)

Reply:
Thanks, updated the style